Gopkg.toml

```diff
@@ -43,7 +47,7 @@ required = [
   name = "github.com/giantswarm/backoff"
 
 [[constraint]]
-  branch = "master"
+  branch = "kubernetes-1.15.4"
```

Comment:
This is to be reverted once https://github.com/giantswarm/helmclient/pull/119 is merged.